tests/compat.sh

```diff
@@ -612,11 +617,21 @@ add_gnutls_ciphersuites()
             ;;
 
         "RSA")
+            # TLS-RSA-WITH-NULL-SHA256 is a (D)TLS 1.2-only cipher suite,
+            # like all SHA256 cipher suites. But Mbed TLS supports it with
+            # (D)TLS 1.0 and 1.1 as well. So does GnuTLS < 3.4.7. In GnuTLS,
+            # supporting this cipher suite in earlier protocol versions was
+            # considered a bug which was fixed in GnuTLS 3.4.7: the server
+            # no longer offers this cipher suite, but the client still does.
```

Comment:
> but the client still does

It does when I test on my local machine, but not when I use the docker image. Investigating... I might end up just skipping both directions.